administrator/language/de-DE/com_config.ini

```
@@ -94,7 +94,7 @@ COM_CONFIG_FIELD_FILTERS_NO_HTML="Kein HTML"
 COM_CONFIG_FIELD_FILTERS_ALLOWED_LIST="Erlaubte Elemente"
 COM_CONFIG_FIELD_FORCE_SSL_DESC="Diese Option erlaubt es, dass bestimmte Bereiche nur noch über HTTPS aufgerufen werden können und sichere Cookies verwendet werden. Diese Bereiche können dann nicht mehr ohne HTTPS aufgerufen werden!<br><br><strong>Hinweis:</strong><br>Beim Webserver bzw. Load Balancer muss das HTTPS-Protokoll aktiviert sein, um diese Option verwenden zu können!<br>„Load Balancer aktivieren“, wenn SSL auf dem Load Balancer terminiert, die Website auf dem Webserver aber per http ausgeliefert wird."
 COM_CONFIG_FIELD_FORCE_SSL_LABEL="HTTPS erzwingen"
-COM_CONFIG_FIELD_GZIP_COMPRESSION_DESC="Sofern vom Webserver unterstützt, wird die Seitenausgabe komprimiert ausgeben (buffered output).<br><br><strong>Hinweis:</strong><br>Joomla! komprimiert dabei nur die generierte HTML-Seite. JavaScript, CSS, Bilder etc. werden <u>nicht</u> komprimiert."
+COM_CONFIG_FIELD_GZIP_COMPRESSION_DESC="Sofern vom Webserver unterstützt, wird die Seitenausgabe komprimiert ausgeben (buffered output).<br><br><strong>Hinweis:</strong><br>Joomla komprimiert dabei nur die generierte HTML-Seite. JavaScript, CSS, Bilder etc. werden <u>nicht</u> komprimiert."
```

```suggestion
COM_CONFIG_FIELD_GZIP_COMPRESSION_DESC="Sofern vom Webserver unterstützt, wird die Seitenausgabe komprimiert ausgeben (buffered output).<br><br><strong>Hinweis:</strong><br>Joomla! komprimiert dabei nur die generierte HTML-Seite. JavaScript, CSS, Bilder etc. werden <u>nicht</u> komprimiert."
```
Nach der immer noch geltenden CI soll Joomla mit `!` geschrieben werden - die Ausnahme sind zusammengesetzte Wörter, wie bei "Joomla-Hilfe". Siehe auch "[korrekte Schreibweise](https://docs.joomla.org/Joomla:Proper_use_of_the_Joomla!_name/de)"